src/ResourceManager/DevTestLabs/Commands.DevTestLabs.Test/ScenarioTests/DevTestLabsTestFixture.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.Azure.Test;
+using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
+using System;
+using System.Threading;
+
+namespace Microsoft.Azure.Commands.DevTestLabs.Test.ScenarioTests
+{
+    public class DevTestLabsTestFixture : RMTestBase, IDisposable
+    {
+        private Random _random = new Random();
+        private string _resourceGroupName;
+        private string _labName;
+        private DevTestLabsController _controller;
+
+        private SynchronizationContext _synchronizationContext;
+
+        public DevTestLabsController Controller
+        {
+            get
+            {
+                Init();
+
+                return _controller;
+            }
+        }
+
+        public void Init()
+        {
+            if (_resourceGroupName != null)
+            {
+                return;
+            }
+
+            _synchronizationContext = SynchronizationContext.Current;
+
+            _controller = DevTestLabsController.NewInstance;
+
+            var callingClassType = TestUtilities.GetCallingClass(1);
+            var mockName = TestUtilities.GetCurrentMethodName(1);
+
+            _controller.RunPsTestWorkflow(
+                () => new[] { "Setup-Test-ResourceGroup " + _resourceGroupName + " " + _labName },
```

Comment:
I am not sure what you mean. We are using Microsoft.Azure.Test.HttpRecorder.HttpMockServer.GetAssetName("DevTestLabsTests", "onesdk"); 

Isn't that common infrastructure? We wanted to make sue that the costly setup only happens once (hence the use of a feature) and that the fixture gets cleaned up. This is the best I could come up with.